resources/js/wayfinder.ts

```diff
@@ -60,7 +60,9 @@ export const queryParams = (options?: {
             });
 
             for (const subKey in query[key]) {
-                params.set(`${key}[${subKey}]`, getValue(query[key][subKey]));
+                if (query[key][subKey] !== undefined) {
+                    params.set(`${key}[${subKey}]`, getValue(query[key][subKey]));
+                }
```

Comment:
I have a love hate relationship with structurally typed languages. This is the _hate_ part.

https://effectivetypescript.com/2020/05/26/iterate-objects/